app/main.py

```diff
@@ -1,3 +1,59 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> int:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> int:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, "Distance"]) -> (int, float):
```

Comment:
Change your type annotation to use `|` instead of `Union`, `Distance` instead of `"Distance"` and `Distance` class instance in the return type annotation. 
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```
In case you face errors, add `from __future__ import annotations` import. Fix type annotation in all places.